postgres/managing/backup-and-restore.html.markerb

```diff
@@ -84,12 +84,27 @@ vs_OPQXXna6kA2Qnhz8 26 MiB 2 days ago
 
 The values under the `ID` columns are what will be used to restore a snapshot.
 
+## Identifying your Postgres image version
+Depending on when you created your Postgres cluster, it may be running an older image than the default for newly created clusters. Different Postgres major versions may not be fully compatible, so it's important to use the same version for your restored cluster.
+
+To see your Postgres image and version, run `fly image show`. 
+
+```cmd
+fly image show -a <postgres-app-name>
+```
+```output
+MACHINE ID      REGISTRY                REPOSITORY      TAG     VERSION DIGEST                        LABELS                                                                          
+e286004f696700  registry-1.docker.io    flyio/postgres  14.6    v0.0.41 sha256:3c25db96357a78e827ca7d fly.app_role=postgres_clusterfly.pg-version=14.6-1.pgdg110+1fly.version=v0.0.41
+```
+The values under the `REPOSITORY and `TAG columns are the image you'll use to restore the snapshot. 
+Legacy postgres images use the `flyio/postgres` repository, while new Postgres Flex images use the flyio/`postgres-flex` repository. In the above example, the machine is running a legacy `flyio/postgres:14.6` image.
+
 ## Restoring from a snapshot
 
-To restore a Postgres application from a snapshot, simply specify the `--snapshot-id` argument when running the `create` command as shown below:
+To restore a Postgres application from a snapshot, simply specify the `--snapshot-id` argument and the `--image-ref` argument when running the `create` command as shown below:
```

Comment:
One has to also specify `--stolon` if it's a Stolon-based cluster (14.x and lower, I think). For a few people who tried a restore, even with the correct image,  without `--stolon` replication is not set up correctly. Running with `--stolon` fixes things.